helm/superset/templates/deployment-worker.yaml

```diff
@@ -31,11 +31,16 @@ spec:
       release: {{ .Release.Name }}
   template:
     metadata:
-      {{ if .Values.supersetWorker.forceReload }}
       annotations:
+        checksum/superset_config.py: {{ include "superset-config" . | sha256sum }}
+        checksum/connections: {{ .Values.supersetNode.connections | toYaml | sha256sum }}
+        checksum/extraConfigs: {{ .Values.extraConfigs | toYaml | sha256sum }}
+        checksum/extraSecretEnv: {{ .Values.extraSecretEnv | toYaml | sha256sum }}
+        checksum/configOverrides: {{ .Values.configOverrides | toYaml | sha256sum }}
```

Comment:
_[K8S newbie]_ What's the point of those annotations?

Reply:
Since those values will change on any content updates of the source values, this will force a patch of the `Deployment`, and a restart of the pods whenever we publish an update to these (otherwise, updates to a `ConfigMap` does not automatically force the pods mounting them to restart).